templates/nginx-conf.j2

```diff
@@ -47,6 +47,12 @@ http {
     sendfile        on;
     #tcp_nopush     on;
 
+    {% if nginx_proxy_conf_http is defined %}
```

Comment:
If you set `nginx_proxy_conf_http: []` in `defaults.yml` you can get rid of the `if` statement.